.build/TornadoVM-Tests.run.xml

```diff
@@ -0,0 +1,30 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="TornadoVM-Tests" type="PythonConfigurationType" factoryName="Python">
+    <module name="tornado-annotation" />
+    <option name="ENV_FILES" value="" />
+    <option name="INTERPRETER_OPTIONS" value="" />
+    <option name="PARENT_ENVS" value="true" />
+    <envs>
+      <env name="BACKEND" value="opencl" />
+      <env name="PYTHONUNBUFFERED" value="1" />
+      <env name="selected_backends" value="opencl-backend" />
+      <env name="TORNADO_SDK" value="$PROJECT_DIR$/bin/sdk" />
+      <env name="JAVA_HOME" value="$PROJECT_DIR$/etc/dependencies/TornadoVM-graal-jdk-21/graalvm-community-openjdk-21.0.1+12.1" />
```

Comment:
What if we don't use the same JDK?

Reply:
It seems to be working, but user must be careful to define the same `JAVA_HOME` when TornadoVM is built and run.